SC_redesign/Crop_and_Soil/soil/phosphorus_cycling/manure.py

```diff
@@ -0,0 +1,316 @@
+from typing import Optional
+from math import exp, sqrt
+
+from SC_redesign.Crop_and_Soil.soil.soil_data import SoilData
+from SC_redesign.Crop_and_Soil.crop_and_soil_constants import MILLIMETERS_TO_CENTIMETERS, KILOGRAMS_TO_GRAMS, \
+    KILOGRAMS_TO_MILLIGRAMS, HECTARES_TO_SQUARE_CENTIMETERS, HECTARES_TO_SQUARE_MILLIMETERS, CUBIC_MILLIMETERS_TO_LITERS
+
+"""
+This module adds and tracks manure phosphorus dynamics based on the SurPhos model.
+"""
+
+
+class Manure:
+
+    def __init__(self, soil_data: Optional[SoilData] = None):
+        """This method initializes the SoilData object that this module will work with, or create one if none provided.
+
+        Parameters
+        ----------
+        soil_data : SoilData, optional
+            The SoilData object used by this module to track manure phosphorus activity, creates new one if one is not
+            provided.
+
+        """
+        self.data = soil_data or SoilData()
+
+    # --- Static Methods ---
+    @staticmethod
+    def _determine_temperature_factor(avg_air_temperature: float) -> float:
+        """Calculates the temperature factor for the current day
+
+        Parameters
+        ----------
+        avg_air_temperature : float
```

Comment:
```suggestion
        mean_air_temperature : float
```

to avoid the abbreviation.